devicon.json

```diff
@@ -904,6 +904,23 @@
         "color": "#1384c8",
         "aliases": []
     },
+    {
+        "name": "dot-net-core",
```

Comment:
We don't use hyphens here. Please change back as how it was.

Reply:
![image](https://user-images.githubusercontent.com/17247763/111661712-8d36d500-880f-11eb-94cd-db22776e27e0.png)

Reply:
@amacado @Thomas-Boi should it be `dot-net-core` or `dotnetcore`?

Reply:
Luckily for us, we have run into this issue before. See this [discussion](https://github.com/devicons/devicon/discussions/459).

I think our plan is future icons can use the dash '-' since there are no issue in terms of our current build/peek script. As long as the name in the `devicon.json` matches the folder name and the individual `svg` name, I think it should be fine.

I'll update the `CONTRIBUTING.md` so this is reflected properly.

Reply:
Regarding the mentioned discussion I would suggest so stick to `dotnetcore` (non dashed)